src/Adapter/MSTest.TestAdapter/MSTestSettings.cs

```diff
@@ -191,6 +191,11 @@ public static RunConfigurationSettings RunConfigurationSettings
     /// </summary>
     public bool TreatClassAndAssemblyCleanupWarningsAsErrors { get; private set; }
 
+    /// <summary>
+    /// Gets a value indicating whether initialize and cleanup methods are added in test results or not.
+    /// </summary>
+    public bool FixturesEnabled { get; private set; }
```

Comment:
What about `ConsiderFixturesAreSeparateEntries`? I would also update the comment to make it more explicit this will create a different test case for the fixture and so it will be reported in UI, TRX and discovered tests (can you check that part?)

Reply:
ReportFixturesToClient? ReportFixtureResultsExplicitly? I think you had one more good word for these somewhere @Evangelink  that was not fixtures. So it is not confused with the Fixtures functionality we wanted for mstest and had in TATF.

Reply:
I am thinking to have this as enum rather than boolean.

```
<FixturesVisibility>Hidden</FixturesVisibility>
<FixturesVisibility>Visible</FixturesVisibility>
<FixturesVisibility>VisibleOnlyOnFailure</FixturesVisibility>
....
```

Reply:
If that's not too complex to handle, then the enum makes sense as it offers some flexibility.

Reply:
I checked the possibility of adding enum. It is not complex, but it will require adding new resources. I would suggest to keep it as it is. There might be some changes related to logging for fixture methods and this can be handled at that time. Maybe it will require to add more settings.

Reply:
I agree, let's go minimal for now.